doc/source/serve/tutorials/batch.md

```diff
@@ -33,9 +33,9 @@ async def my_batch_handler(self, requests: List):
     pass
 ```
 
-This batch handler can then be called from another `async def` method in your deployment.
-These calls will be batched and executed together, but return an individual result as if
-they were a normal function call:
+Ray Serve can then call the batch handler from another `async def` method in your deployment.
```

Comment:
Ray Serve doesn't call this handler. The user's code calls it. I believe the old sentence was trying to say "You can call this batch handler from another `async def` method in your deployment."

I'm not sure how to express this without saying "You can" though. Maybe we can just drop that phrase? "Call this batch handler from another `async def` method in your deployment."

Reply:
reverted